### ISSUE TITLE
Deprecated? 

### ISSUE THREAD

**Issue by httpdss** [shizzz477/msploitego#11]
just wondering is this great project deprected? is there any work going on? didnt find any newer fork but just wondering.

**Comment by shizzz477** [comment]
yes it is unfortunately...too busy with my day job.  glad you liked it, it was fun creating it
